lib/zipkin-tracer/config.rb

```diff
@@ -16,7 +16,7 @@ def initialize(app, config_hash)
       # The name of the current service
       @service_name      = config[:service_name]
       # The port where the current service is running
-      @service_port      = config[:service_port]      || DEFAULTS[:service_port]
+      @service_port      = config[:service_port]
```

Comment:
I dont thing we need this at all. This is only there for local and we are not sending it now, right?